### DISCUSSION TITLE
Dependencies in DAX

### DISCUSSION THREAD

**Discussion by janhoran** [TabularEditor/TabularEditor3#1016] [🙋♀️TE3 Q&A] [answered]
Hello, 

The dependency feature is great! However, I'd like to have a document of ALL dependencies of measures. How can we do that? Is it possible?

**Comment by otykier** [comment, marked as answer]
You can use C# scripting to export dependencies in various ways. Here is an example: https://docs.tabulareditor.com/te2/Useful-script-snippets.html#exporting-dependencies-between-tables-and-measures
